test/unit/mdc-menu/mdc-menu.test.js

```diff
@@ -39,10 +39,37 @@ function getFixture(open) {
         <li tabIndex="-1" class="mdc-list-item" role="menuitem">Item</a>
         <li role="separator"></li>
         <li tabIndex="-1" class="mdc-list-item" role="menuitem">Another Item</a>
-        <ul class="mdc-menu__selection-group mdc-list" role="menu">
-          <li tabIndex="-1" class="mdc-list-item" role="menuitem">Item</a>
-          <li tabIndex="-1" class="mdc-list-item mdc-menu-item--selected" role="menuitem">Another Item</a>
-        </ul>
+        <li>
```

Comment:
Do you think we should create new fixture for single selection group as well?

Reply:
That is what the `getFixture` method already has. Or did you want to isolate the selection group?

Reply:
Ah ok. Yes we can keep this change if selection group is already in the fixture.